### PR TITLE
refactor toolkit toolwindow reload utils

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ToolkitUtils.kt [hunk 1]
```diff
@@ -41,3 +46,32 @@ fun inspectExistingConnection(project: Project): Boolean =
...
+
+    val contentManager = AwsToolkitExplorerToolWindow.toolWindow(this).contentManager
+
+    val myContent = contentManager.factory.createContent(component, null, false).also {
+        it.isCloseable = true
+        it.isPinnable = true
+    }
+
+    runInEdt {
+        contentManager.removeAllContents(true)
```
Comment (rli) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ToolkitUtils.kt:74]: do we need to dispose the content or does that happen automatically

Comment (Will-ShaoHua) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ToolkitUtils.kt:74]: yea I feel we should do this

Comment (Will-ShaoHua) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ToolkitUtils.kt:74]: apparently I have not done this part yet (the entire browserPanel is not dispoable yet)

Comment (Will-ShaoHua) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ToolkitUtils.kt:74]: lol

```
java.lang.RuntimeException: Memory leak detected: 'Module: 'Calculator'' (class com.intellij.workspaceModel.ide.impl.legacyBridge.module.ModuleBridgeImpl) was registered in Disposer as a child of 'ROOT_DISPOSABLE' (class com.intellij.openapi.util.Disposer$2) but wasn't disposed.
Register it with a proper parentDisposable or ensure that it's always disposed by direct Disposer.dispose call.
See https://jetbrains.org/intellij/sdk/docs/basics/disposers.html for more details.
The corresponding Disposer.register() stacktrace is shown as the cause:
```

**Pull request closed without merging**